getting-started/installation-homebrew.md

```diff
@@ -27,7 +27,7 @@ brew install timescaledb
 /usr/local/bin/timescaledb_move.sh
 ```
 
-#### Update Postgresql.conf
+#### Update postgresql.conf
```

Comment:
Should we put the filename in backtick quotes?  Or maybe change it completely to "Update PostgreSQL config"?